site.yml

```diff
@@ -14,6 +14,18 @@
   - { role: base }
   - { role: serf }
   - { role: dev }
+  - { role: vagrant }
+
+- hosts: build
```

Comment:
@erikh 

> What is getting pulled in that we need to keep it for?

things like etcd, ovs, docker, docker-compose, skydns and collins container images etc come from `dev` role that we need in packer images.

> Let's separate anything out that's not necessary and ensure it doesn't get into build, e.g. they can't even use packer or vbox.

yep, that is what I meant too. In the gist/commit that I shared I have commented out the tasks that will install vagrant, packer and vxox. I think we can move them out for now to get our build images sane. I can polish it a bit and submit a patch if you think this sounds ok.